packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -135,6 +135,15 @@ class ScrollableBaseNative extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Prevents scroll by wheeling on the list or the scroller.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @private
+		 */
+		noScrollHorizontalByWheel: PropTypes.bool,
```

Comment:
What about `noHorizontalScrollByWheel` or `noScrollHorizontallyByWheel`? :)

Reply:
It looks better. :) But I want a much better name!

Reply:
We need @webOS101 's opinion!